base4kids2.ldif

```diff
@@ -498,6 +498,43 @@ attributeTypes: (
       DESC 'Determines whether a base4kids object is disabled but still active'
       SUP base4kidsIsBlocked )
 #
+# Name:     base4kidsTelephoneNumberBusiness
```

Comment:
Please rename the `base4kidsTelephoneNumberBusiness` attribute  to `base4kidsBusinessTelephoneNumber` to follow the same naming convention as `homeTelephoneNumber` (RFC 1274) and `facsimileTelephoneNumber` (RFC 2256)